doc/user/content/sql/create-index.md

```diff
@@ -1,34 +1,35 @@
 ---
 title: "CREATE INDEX"
-description: "`CREATE INDEX` creates an in-memory index on a view."
+description: "`CREATE INDEX` creates an in-memory index on a source or view."
 menu:
   main:
     parent: 'sql'
 ---
 
 {{< warning >}} This is an advanced feature of Materialized; most users will not
 need to manually create indexes to maximize the value Materialize offers, as
-running `CREATE MATERIALIZED VIEW` automatically creates all required indexes to
-eagerly materialize that view. {{< /warning >}}
+running `CREATE MATERIALIZED SOURCE` or `CREATE MATERIALIZED VIEW` automatically
+creates an index which will eagerly materialize that source or view. {{< /warning >}}
 
-`CREATE INDEX` creates an in-memory index on a view.
+`CREATE INDEX` creates an in-memory index on a source or view.
 
 - For materialized views, this creates additional indexes.
 - For non-materialized views, it converts them into materialized views.
 
 ## Conceptual framework
 
-Indexes persist some subset of a query's data to memory, which let materialized
-views quickly perform reads from materialized views, which includes performing
-[`JOIN`](../join)s. For more information, see [API Components:
-Indexes](../../overview/api-components#indexes).
+Indexes assemble and maintain in memory the data that results from a query, which can
```

Comment:
I think the reason it sounds weird to me is that I know (maybe the reader does not) that a query is producing a stream of updates, rather than a singular result. I'm happy to just change the text, but I tried in several spots to make it clear that an index wasn't a snapshot, and I think this was one of those places.

Reply:
change made (but let's keep eyes open for language that works well for standard SQL folks and "data may change" folks).

Reply:
Ah, I can see what you mean here. Certainly tricky.

Reply:
If nothing else, it should have been "data that result". :D